### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by tohrnii**
Addressing #16.
Adds basic end to end integration tests for AIR DSL.
This PR should be merged after #41.

### CODE REVIEW

**Review by grjte — CHANGES_REQUESTED** [review]
This is great! I left one nit inline, but otherwise we can merge these once the other PR is merged. Thank you!

File: air-dsl/Cargo.toml
Comment (grjte) [inline air-dsl/Cargo.toml:15]: actually, one more note - are these imports actually required here? It doesn't seem like it, since we're not actually actually using the generated rust files which contain the imports that would require these dependencies

Comment (tohrnii) [reply air-dsl/Cargo.toml:15]: I was using them to test for the other PR. Will remove them.

File: air-dsl/tests/periodic_columns/periodic_columns.rs
Comment (grjte) [reply air-dsl/tests/periodic_columns/periodic_columns.rs:55]: I added #48

**Review by grjte — CHANGES_REQUESTED** [review]
This looks good except that I'm seeing changes in the `codegen-winter` crate which were part of the other PR. Can you just update so we only have the `air-dsl` and `examples` changes and also clean up the commit history? Then we can merge

**Comment by tohrnii** [comment]
> This looks good except that I'm seeing changes in the codegen-winter crate which were part of the other PR. Can you just update so we only have the air-dsl and examples changes and also clean up the commit history? Then we can merge

Done. I was just waiting for the other PR to be merged.

**Review by grjte — APPROVED** [review]
Looks great, thanks again!

**Pull request merged**
